integration_test/ops_agent_test.go

```diff
@@ -80,7 +80,21 @@ func configPathForPlatform(platform string) string {
 
 func restartCommandForPlatform(platform string) string {
 	if gce.IsWindows(platform) {
-		return "Restart-Service google-cloud-ops-agent -Force"
+		return `
+Restart-Service google-cloud-ops-agent -Force
+if (!$?) {
```

Comment:
consider a TODO to remove this once the bug  is fixed. optional.

Reply:
Added